apps/src/templates/instructions/InstructionsWithWorkspace.jsx

```diff
@@ -93,10 +111,35 @@ export class UnwrappedInstructionsWithWorkspace extends React.Component {
     window.removeEventListener('resize', this.onResize);
   }
 
+  /**
+   * Given a prospective delta, determines how much we can actually change the
+   * height (accounting for min/max) and changes height by that much.
+   * @param {number} delta
+   * @returns {number} How much we actually changed
+   */
+  handleHeightResize = delta => {
+    const {
+      instructionsHeight: oldHeight,
+      instructionsMaxHeight: maxHeight,
+      setInstructionsRenderedHeight: setHeight
+    } = this.props;
```

Comment:
cool syntax trick - I've never seen this before!

Reply:
Yeah! I find it useful sometimes, although it's not always the most intuitive read.  For more details, see [MDN / Destructuring assignment / Object destructuring / Assigning to new variable names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variable_names).